sklearn/tests/test_multioutput.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+
+from sklearn.base import clone
+from sklearn.ensemble import RandomForestClassifier
+from sklearn import datasets
+from sklearn.multioutput import MultiOutputClassifier
+from sklearn.preprocessing import LabelBinarizer
+from sklearn.utils import shuffle
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_almost_equal
+
+
+def _create_data_set():
+    # Creates a multi-target data set using the iris data set
+    # X : The iris predictor data.
+    # Y : A multi-target(150x3) array generated from the original response data
+
+    # Import the data
+    iris = datasets.load_iris()
+    X = iris.data
+
+    # create a multiple targets by randomized shuffling and concatenating y.
+    y1 = iris.target
+    y2 = shuffle(y1, random_state=1)
+    y3 = shuffle(y1, random_state=2)
+
+    # concatenate the array and transpose
+    Y = np.vstack((y1, y2, y3)).T
+
+    return(X, Y)
+
+
+def _set_up_multi_target_random_forest():
+    # Set up the forest and multi-target forest'''
+
+    forest = RandomForestClassifier(n_estimators=100, random_state=1)
+    multi_target_forest = MultiOutputClassifier(forest, n_jobs=-1)
+
+    return forest, multi_target_forest
+
+
+def test_multi_target_fit_with_random_forest():
+    # test if multi_target initializes correctly with random forest and fit
+    # assert predictions work as expected for predict, prodict_proba and score
+
+    X, Y = _create_data_set()
```

Comment:
The function ``_create_data_set`` is actually using the iris dataset from ``datasets`` module.